src/io.ts

```diff
@@ -12,20 +15,49 @@ export const startIOServer = (httpServer: ServerType) => {
     },
   });
 
+  const db = new DbInterface();
+
+  let questions: Array<string> = [];
+  let recvQuestion = 0, playerCount = 0, questionIndex = 0;
+
+  function sendQuestionToPlayers(roomId: string) {
```

Comment:
extract this function into another file meant for helper functions in the `io` folder (you will need to make one)